test/utils/cleandb.sh

```diff
@@ -5,7 +5,7 @@ dropKeyspaces ( ) {
   then
     PATTERN=$1
     echo "looking for keyspaces named '*$PATTERN*'..."
-    for KEYSPACE in `echo 'select keyspace_name from system.schema_keyspaces;' | cqlsh | grep $PATTERN`
+    for KEYSPACE in `echo 'select keyspace_name from system.schema_keyspaces;' | cqlsh `
```

Comment:
Why has the `grep` been removed? Now this is dropping all of the keyspaces in Cassandra.